examples/r2i/ncsdk/tinyyolo.cc

```diff
@@ -47,17 +132,22 @@ void PrintTopPrediction (std::shared_ptr<r2i::IPrediction> prediction,
    *    [980:1078]: 7*7*2 probability multiplicator for each box in the grid
    *    [1078:1470]: 7*7*2*4 [x,y,w,h] for each box in the grid
    */
-  int i, j, c, b;
   r2i::RuntimeError error;
+
+  int i, j, c, b;
+  int box_probs_start = GRID_H * GRID_W * CLASSES;
+  int all_boxes_start = GRID_H * GRID_W * CLASSES + GRID_H * GRID_W * BOXES;
+  int index;
+
   double class_prob;
   double box_prob;
   double prob;
-  double max_prob = 0;
-  int max_prob_row = 0;
-  int max_prob_col = 0;
-  int max_prob_class = 0;
-  int max_prob_box = 0;
-  double box[4];
+  double iou;
+
+  box result;
+  std::list<box> boxes;
+  std::list<box>::iterator it1;
+  std::list<box>::iterator it2;
 
   std::string labels [CLASSES] = {"aeroplane", "bicycle", "bird", "boat",
```

Comment:
This is highly dependent on how the model was trained. At least, I would leave a comment saying so.